plugins-root/check_icmp.c

```diff
@@ -591,6 +592,14 @@ int main(int argc, char **argv) {
         }
         break;
 
+      case 'f':
+        perfdata_sep = optarg;
+        break;
+
+      case 'F':
+        perfdata_num = strtoul(optarg, NULL, 0);
```

Comment:
perfdata_num was initialized as an `int`. I think `strtod()` is more appropriate here

Reply:
`strtod()` converts a string to double, `strtoul()` converts to unsigned long.
Note that `timeout` (for example) is also declared as an `int`, and use `strtoul()` a few lines below.
I then went the same way.